integration_test/third_party_apps_data/agent/ops-agent/linux/supported_applications.txt

```diff
@@ -11,3 +11,5 @@ postgresql
 redis
 solr
 tomcat
+elasticsearch
```

Comment:
I guess the `elasticsearch` diff will go away after rebasing?

Reply:
Oops good call I didn't realize that had been inserted above